### ISSUE TITLE
error when bungeecord enabled

### ISSUE THREAD

**Issue by ehamawy** [LOOHP/Limbo#60]
When bungeecord=true in server.properties, clients will display this error, and are unable to connect.
![jlsdfjlsdaf](https://user-images.githubusercontent.com/9953712/217760606-0b5410cd-b41e-4fa4-bd46-70610c6579c9.PNG)

**Comment by Timongcraft** [comment]
This is true I think it is because the bungeecord value isn't checked correctly because in the console it says:

![image](https://user-images.githubusercontent.com/77288029/227766924-f880a316-7fa2-4920-b00a-cc82b3ac2d2e.png)

**Comment by Timongcraft** [comment]
Correction only occurs if you have bungee-guard enabled

**Comment by Timongcraft** [comment]
But the packet error is still there

**Comment by Timongcraft** [comment]
Hello?

**Comment by Timongcraft** [comment]
I found the issue you need to delete the Internal_Data folder

**Comment by TheGallium** [comment]
Where is the `Internal_Data` folder?

**Comment by Timongcraft** [comment]
It doesn't exist anymore
